api/app/src/models/medical/organization.ts

```diff
@@ -12,10 +11,19 @@ export enum OrgType {
   postAcuteCare = "postAcuteCare",
 }
 
+export type OrganizationAddress = {
+  addressLine1: string;
+  addressLine2?: string;
+  city: string;
+  state: string;
+  zip: string;
+  country: string;
+};
```

Comment:
So we don't have to rewrite what an Address is composed of, we could use TS' utility types, something like:

```ts
export type OrganizationAddress =
    Pick<Address, "addressLine2"> &
    Required<Omit<Address, "addressLine2">>;
```